wgsl/extract-grammar.py

```diff
@@ -530,7 +530,8 @@ def not_token_only(value):
     if "function-scope" in key:
         value = ["fn function__scope____() {"] + value + ["}"]
     if "type-scope" in key:
-        value = ["let type_scope____: "] + value + [";"]
+        # Intiailize with zero-value expression.
```

Comment:
`Intiailize` -> `Initialize` (in case the first one is not a term that I am unaware of)

Reply:
Thanks, found some others in the spec with the same misspelling.

Reply:
@alan-baker great, thank you for the fixes!